package.json

```diff
@@ -4,7 +4,7 @@
   "description": "exercises for learning Github in Digital History",
   "main": "index.js",
   "scripts": {
-    "test": "mocha -t 0 --reporter mochawesome --reporter-options reportDir=TestResults,reportFilename=testresults || true",
```

Comment:
this is because i had to fix the mocha timeout error, tried to revert it back to the master version here, but am not entirely sure if it worked.

Reply:
just a whitespace error, which you can see a little bit in this view.  no big deal, but good to learn to spot these errors.